PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+### IMPORTANT: Please do not create a Pull Request without creating an issue first.
+
+### Title
+Please remove this section and [use conventional commits structure](https://www.conventionalcommits.org/en/v1.0.0/) for PR titles.  
+E.g. feat(api): add Foo to Bar
+
+### Description
+A one to two sentence summary of the PR for easy digestion, must be different and more descriptive than title. Combination of title and description should summarize the issue + solution.
+
+### Reviewers
+List all reviewers here and briefly describe why they are invited for review.
+By default, the PR author should merge once the reviewers have had a chance to review.
+If you have a different preference or cannot merge after review for some other constraint, leave a note about it here.
+
+### Changes
+List your changes
+
+### Acceptance Criteria
+What needs to happen functionally for this work to pass review?  
+
+### Related Issue(s)
+Remove this line and link your issue here: Issue #1234 - Add Foo to Bar.  
```

Comment:
Could we please use the magic words that GitHub recognizes to automatically close issues, like `Closes #NNNN` or `Fixes #NNNN`?

Reply:
Good catch! yes, we should do this. I'll add a commit to this PR